ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86.yaml

```diff
@@ -22,7 +22,7 @@ resources:
       memory: 200Mi
 tests:
 - as: perfscale-aws-control-plane-120nodes-periodic
-  cron: 0 13 * * 4
+  cron: 0 14 * * 4
```

Comment:
should we change this to a different day so we can have 4.15 run on thursday for the 4.next testing?

Reply:
sorry, not sure I am understanding. 
It is running on Thursday right? 
Also we wanted to be able to compare results across workloads for versions, providers etc.. so tried to keep all on Thu based on that but spread out.

Reply:
At the end of each sprint a 4.15 job is selected to be put out as an "ec" release. That I feel should take precedence in running on Thursdays and could move other tests to different days